i18n/fr/docusaurus-plugin-content-docs/version-22.10/graph-views/map-web-upgrade.md

```diff
@@ -69,13 +101,24 @@ sudo yum update centreon-map-engine centreon-map-web-client
 </TabItem>
 </Tabs>
 
-2. Assurez-vous d'utiliser la bonne version de MariaDB et mettez-la à jour si besoin. Voir [Mettre à jour MariaDB](../upgrade/upgrade-mariadb.md).
-
-## Mise à jour de l'interface
+3. Videz le cache de votre navigateur.
 
-Se connecter à l'interface web de Centreon et se rendre dans le menu
-`Administration > Extensions > Gestionnaire`.
+4. Finalisez la mise à jour du module et du widget dans l'interface Centreon **Administration > Extensions > Gestionnaire**.
 
-Un bouton orange de mise à jour est visible et signale qu'une mise à
+ > Un bouton orange de mise à jour est visible et signale qu'une mise à
 jour est disponible, cliquez dessus pour mettre à jour le module, faire
 de même pour le widget.
+
+5. Redémarrez le service **centreon-map-engine** en exécutant la commande suivante :
+ 
+  ```shell
+  sudo systemctl start centreon-map-engine
+  ```
+
+6. Si vous utilisez MAP Legacy, entrez cette commande sur le serveur MAP pour reporter une éventuelle personnalisation.
+
+ ```shell
+ diff -u /etc/centreon-studio/centreon-map.conf /etc/centreon-studio/centreon-map.conf.rpmsave
```

Comment:
This command only works for EL7/EL8/EL9 but not on Debian.
Debian will not create .rpmsave files
Also, if we update Centreon Map, we don't care about /etc/centreon-studio/ folder because it's related to centreon Map Legacy and the update will not modify these files